src/test/java/net/snowflake/client/jdbc/CustomProxyLatestIT.java

```diff
@@ -415,7 +417,10 @@ public void testProxyConnectionWithoutProxyPortOrHost()
             + "&proxyUser=testuser1&proxyPassword=test"
             + "&useProxy=true";
     try {
-      runAzureProxyConnection(connectionUrl, false);
+      runAzureProxyConnection(
+          connectionUrl, /* usesConnectionProperties */
```

Comment:
can the comment on the same line?

Reply:
the formatting put it on different lines! This is the only way it passes the format check.